src/server/server.js

```diff
@@ -32,42 +37,25 @@ mongoose
   )
   .catch((err) => log(serverLog(err)));
 
-// **** middleware ****//
-app.use((req, res, next) => {
-  // Authorizing API call to come from React front end on port 3000
-  res.setHeader("Access-Control-Allow-Origin", "http://localhost:3000");
-  res.setHeader(
-    "Access-Control-Allow-Methods",
-    "GET, POST, OPTIONS, PUT, PATCH, DELETE"
-  );
-  res.setHeader(
-    "Access-Control-Allow-Headers",
-    "X-Requested-With,content-type"
-  );
-  res.setHeader("Access-Control-Allow-Credentials", true);
-  next();});
+app.use(cors({
+  origin: 'http://localhost:3000', 
+  credentials: true
+}))
 app.use(express.urlencoded({extended: true,}));
 app.use(express.json());
-app.use(session({secret: "secret"})); // Set up express-sessions
+app.use(session({ //setup session middleware
+  secret: "secret",
```

Comment:
Guess this has to be a cyrto secure string.

Consider a 32 length random hexadecimal.

Store in an env var e.g. process.env.SESSION_SIGNING_SECRET

Reply:
Will pop this in an issue for future us 👍